idf_component.yml

```
@@ -0,0 +1,9 @@
+# Metadata for using libcanard as an ESP-IDF component.
+
+# Note: Version is not specified in this file,
+# because the 'esp_publish.yml' GitHub action
+# automatically sets it based on the release version.
+
+description: "OpenCyphal for embedded systems."
```

```suggestion
description: "Cyphal/CAN for embedded systems."
```

Cyphal comes with various transports, CAN is only one of them.